app/2.0/docs/tools/polymer-json.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "`polymer.json` specification"
+---
+
+<!-- toc -->
+
+# About `polymer.json`
```

Comment:
Like the other doc, start with H2s. Also, don't really need a header right after the TOC, I think? We usually just start with introductory text.